ui/components/app/transaction-decoding/components/ui/accreditation/accreditation.component.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getAccountLink } from '@metamask/etherscan-link';
+import {
+  getCurrentChainId,
+  getRpcPrefsForCurrentProvider,
+} from '../../../../../../selectors';
+import { I18nContext } from '../../../../../../contexts/i18n';
+
+import Button from '../../../../../ui/button';
+import Tooltip from '../../../../../ui/tooltip';
```

Comment:
Need to import Typography component and helper object and remove Tooltip

```suggestion
import { TYPOGRAPHY } from '../../../../../../helpers/constants/design-system';

import Button from '../../../../../ui/button';
import Typography from '../../../../../ui/typography';
```